src/views/manage/ManageForm.jsx

```diff
@@ -138,20 +149,47 @@ const ManageForm = ({ id }) => {
               )
             )}
           </ControlsColumn>
-        </Row>
-        <Row>
+          <ControlsColumn
+            span={{ small: 1, medium: 2, big: 6, large: 6, xLarge: 6 }}
+          >
+            <Heading $as="h3" color="secondary">
+              Documenten
+            </Heading>
+            <FileInput
```

Comment:
Should be a self-closing component; doesn't have children.

Reply:
ok!